src/state-display/state-display.ts

```diff
@@ -77,7 +83,10 @@ class StateDisplay extends LitElement {
     const stateObj = this.stateObj;
     const domain = computeStateDomain(stateObj);
 
-    if (content === "state") {
+    if (
+      content === "state" ||
+      (this.format && TIMESTAMP_RENDERING_FORMATS.includes(this.format))
```

Comment:
I don't think we can do this, as relative time is also used for things like `last_triggered`, etc.

Reply:
Yeah, it was something I threw in somewhat last minute when I saw the alternative `state_content` timestamps weren't respecting the format. Would adding more specific guards for the other state content types be an acceptable alternative?

Reply:
I can't recall the specifics surrounding this, but I believe it had something to do with secondary info. Regardless, without that extra information I can't properly defend the change 😆, so I've removed this in the latest commit.